provider-tutorials/provider-cli.md

```diff
@@ -214,13 +214,15 @@ For `--network`you have two options, either `mainnet` or `rinkeby`. For `--to-ad
 
 `ya-provider` allows to fine-tune the config created with `golemsp settings` using commands `config`, `preset`, `profile`, and `exe-unit`.
 
-Additionally, it enables configuration of the certificate keystore and of the domain whitelist, which are used to control what kind of outbound traffic is allowed out of the VM containers run by the requestors on your machine.
+Additionally, it enables configuration of the rules, certificate keystore and domain whitelist, which are used to control what kind of outbound traffic is allowed out of the VM containers run by the requestors on your machine.
 
 ### Keystore
 
-The provider has an embedded certificate keystore which is used to validate any additional permissions for the payload launched by the requestors.
+The provider has an embedded certificate keystore which is used to keep certificates which then can be used by specific Rules.
```

Comment:
I believe this sentence carries no meaning after the change...

Reply:
What do you propose then?
This sentence says exactly what keystore now does: it only gathers all the certificates inside and nothing more. Then it can be used by specific rules to control i.e. outbound

Reply:
maybe something along the lines of:

```suggestion
The provider has an embedded keystore, used to hold certificates. Specific rules can then reference those certificates to enable certain actions.
```